worker/runner.ts

```diff
@@ -8,6 +8,7 @@ import { RoutineJobsTable } from "../db/schema";
 import { eq } from "drizzle-orm";
 import { AgentServer } from "./agent-server";
 import { AGENT_SERVER_PORT } from "../server/constants";
+import { BrowserManager } from "../browser/src/services/BrowserManager";
```

Comment:
Vercel build is failing with the following error

![image](https://github.com/user-attachments/assets/01d2e75a-ea7d-43e6-9f0c-4a1660b1db40)

Why? Either we should have worked and browser both in scope or both out of scope when building vercel lambdas